server/src/main/java/org/opensearch/node/Node.java

```diff
@@ -893,6 +893,7 @@ protected Node(
 
             final RestController restController = actionModule.getRestController();
 
+            List<TransportInterceptor> coreTransportInterceptors = new ArrayList<>();
```

Comment:
nit: List.of(admissionControlTransportInterceptor)

Reply:
Updated.